avocado/utils/memory.py

```diff
@@ -320,3 +320,16 @@ def get_thp_value(feature):
         return (re.search(r"\[(\w+)\]", value)).group(1)
     else:
         return value
+
+
+def memtotal_sys():
+    """
+    get the total memory as detected by system
```

Comment:
@ldoktor actually, I am hitting at issue while having multiple files to process, seems bash has some limit, working out alternatives.